ocamlbuild_plugin/ocamlbuild_cppo.ml

```diff
@@ -1,23 +1,22 @@
 
 open Ocamlbuild_plugin
 
+let insert_cppo_rules ext =
```

Comment:
I was hesitant about making that change but I had 2 justifications:
1. I wanted to stress that it was doing something via side-effect (hence `insert`).
2. I wanted to differentiate the outer function from the inner `cppo_rule` function.

Your point about the `Ocamlbuild.rule` analogy is good, but I find that API so parochial and confusing that to me it is almost an "anti"-point. Regardless, I'm not wedded to name so if you think that `cppo_rules`'s is a better name, I'll gladly change it.